app/controllers/organizations_controller.rb

```diff
@@ -65,17 +62,21 @@ def destroy
   private
   # Use callbacks to share common setup or constraints between actions.
   def set_organization
-    if (Organization.find(params[:id]).users.ids.include? current_user.id)
-      @organization = Organization.find(params[:id])
+    @org_to_check = Organization.find(params[:id])
```

Comment:
do you need `org_to_check` to be a instance variable? if you don't use it in views, then I believe than not.